sys-devel/rust-std/rust-std-1.80.0.ebuild

```
@@ -0,0 +1,155 @@
+# Copyright 2020-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{9..12} )
+
+inherit flag-o-matic multiprocessing python-any-r1 rust-toolchain toolchain-funcs
+
+DESCRIPTION="Rust standard library, standalone (for crossdev)"
+HOMEPAGE="https://www.rust-lang.org"
+SRC_URI="https://static.rust-lang.org/dist/rustc-${PV}-src.tar.xz"
+
```

Move S= straight after SRC_URI

```suggestion
SRC_URI="https://static.rust-lang.org/dist/rustc-${PV}-src.tar.xz"
S="${WORKDIR}/${P/-std/c}-src"

```

---

https://github.com/gentoo/gentoo/compare/1f47d0e041949cdcac80a3dbd66cf7d0b25fc0cc..b28586e5ea267362fef858f215b35f63fa9d113d